README.md

```diff
@@ -455,6 +455,12 @@ m = Money.from_cents('123', :gbp) # => #<Money fractional:123 currency:GBP>
 m.format(symbol: m.currency.to_s + ' ') # => "GBP 1.23"
 ```
 
+If you would like to customise currency symbols (e.g to avoid ambiguous between currencies), you can set custom symbol for specific currency as following
+
+```ruby
+  Money::Currency.table[hkd][:symbol] = 'HK$'
```

Comment:
hello, this doesn't work on the latest Money. Also, `hkd` variable is not defined

Reply:
Hi @tagliala 
Thanks! I forgot to add `:` to the symbol. It's working now. I already used it with latest money version.
 
<img width="514" alt="Screenshot 2024-05-31 at 12 47 26" src="https://github.com/RubyMoney/money/assets/2104202/734e3321-407d-401b-a72a-69dadd264203">

Reply:
01110786525